types/src/front/lib/assistant.ts

```diff
@@ -279,7 +285,22 @@ export const O1_MODEL_CONFIG: ModelConfigurationType = {
     "OpenAI's reasoning model designed to solve hard problems across domains (Limited preview access).",
   shortDescription: "OpenAI's reasoning model.",
   isLegacy: false,
-  supportsVision: false,
+  supportsVision: true,
+};
+export const O1_HIGH_REASONING_MODEL_CONFIG: ModelConfigurationType = {
```

Comment:
I'm making it a separate "model config", because will be adding the O1 models to assistant builder. I don't want to expose configuring the `reasoning_effort` from the UI for now, so it's simpler to expose as a separate model